manifests/nodejs.yml

```
@@ -152,11 +152,13 @@ tests/:
         test_kafka_key.py:
           TestKafkaKey:
             '*': missing_feature
-            express4: missing_feature
+            express4: v5.4.0
+            express4-typescript: v5.4.0
         test_kafka_value.py:
           TestKafkaValue:
             '*': missing_feature
-            express4: missing_feature
+            express4: v5.4.0
```

Why not:
```suggestion
            '*': v5.4.0
            nextjs: irrelevant # nextjs is not related with kafka
```

I think that it gives better information in less lines